### ISSUE TITLE
Update modrinth versions

### ISSUE THREAD

**Issue by Merith-TK** [Ladysnake/Illuminations#116]

**Comment by Fauli1221** [comment]
yeah, now that the Curseforge public API has shut down it would be a blessing to have the mod up-to-date on Modrinth

**Comment by doctor4t** [comment]
I'll start posting regularly on Modrinth once they get around to adding their planned creator rewards program

**Comment by Fauli1221** [comment]
Modrinth has implemented creator payouts https://blog.modrinth.com/creator-monetization-beta
